app/src/main/java/org/mozilla/webmaker/WebmakerActivity.java

```diff
@@ -32,6 +35,15 @@ public WebmakerActivity(String pageName, int id, int layoutResID, int menuResId)
     protected void onCreate(Bundle savedInstanceState) {
         routeParams = new JSONObject();
 
+        SharedPreferences userSession = getSharedPreferences(WebAppInterface.USER_SESSION_KEY, 0);
```

Comment:
This is going to cause a fairly sizable performance hit `onCreate` of each Activity. I'd rather isolate this to just the `Play` activity.